lib/afc/index.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable promise/prefer-await-to-callbacks */
+import LengthBasedSplitter from '../util/transformer/length-based-splitter';
+import { Operations, operationCode, Errors, errorCode } from './protocol';
+import AfcEncoder from './transformer/afcencoder';
+import AfcDecoder from './transformer/afcdecoder';
+import B from 'bluebird';
+
+const AFC_SERVICE_NAME = 'com.apple.afc';
+
+const NULL_DELIMETER_CODE = 0x00;
+
+class AfcService {
+
+  constructor (socketClient) {
+    this._socketClient = socketClient;
+    this._splitter = new LengthBasedSplitter(true, 1000000, 8, 8, -8);
```

Comment:
I still like constants much more than magic values

Reply:
I will change the 1000000 to a constant but I think the rest is not necessary